openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -37,3 +37,17 @@
     'enable_pwned_password_api',
     __name__
 )
+
+
+# .. toggle_name: REDIRECT_TO_LMS
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Set this to True if you want to cms-admin login from lms login.
+# .. in case of logout it will use lms logout also.
```

Comment:
```suggestion
# .. toggle_description: Set this to True if you want to redirect cms-admin login to lms login.
#   In case of logout it will use lms logout also.
```
Wording improvements; use correct line continuation